alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/AggregationTrigger.kt

```diff
@@ -41,8 +42,8 @@ data class AggregationTrigger(
     override val id: String = UUIDs.base64UUID(),
     override val name: String,
     override val severity: String,
+    val bucketSelector: BucketSelectorExtAggregationBuilder,
     override val actions: List<Action>
-    // TODO: Add the remaining member variables based on integrated with Trigger pipeline aggregator
 ) : Trigger {
 
     // TODO: Once class is full implemented add tests to the following suites:
```

Comment:
Could you add a test called `'test aggregation trigger as stream'` in `WriteableTests.kt` (it will be very similar to `'test traditional trigger as stream'`. Then we can remove this `TODO`.